doc/users/prev_whats_new/dflt_style_changes.rst

```diff
@@ -1005,7 +1005,7 @@ a cleaner separation between subplots.
 
            ax = fig.add_subplot(2, 2, j)
            ax.hist(np.random.beta(0.5, 0.5, 10000), 25, density=True)
-           ax.set_xlim([0, 1])
+           ax.set_xlim((0, 1))
```

Comment:
In cases like these, you could drop the parentheses, and pass as two arguments instead.

Reply:
We may even consider discouraging passing as single tuple. If you have a limit tuple, you can nowadays use tuple unpacking `set_xlim(*lims)`.